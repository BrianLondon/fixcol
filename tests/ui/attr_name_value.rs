use fixed_derive::ReadFixed;

#[derive(ReadFixed)]
struct Thing {
    #[fixed = "path/goes/here"]
    field: String,
}

pub fn main() {}
