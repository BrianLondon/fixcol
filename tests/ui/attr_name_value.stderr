error: fixcol-derive error: Could not read config from name/value style attribute.

       Expected parameters like #[fixcol(width = 4)]
 --> tests/ui/attr_name_value.rs:5:16
  |
5 |     #[fixcol = "path/goes/here"]
  |                ^^^^^^^^^^^^^^^^
